pkg/inventory/inventory-client.go

```diff
@@ -160,7 +160,7 @@ func (cic *ClusterInventoryClient) Replace(localInv InventoryInfo, objs []object
 	if err != nil {
 		return err
 	}
-	if object.SetEquals(objs, clusterObjs) {
+	if object.IsSetEqual(objs, clusterObjs) {
```

Comment:
Remove this change. If you want to advocate for a name change, please do it in another PR.